{"uuid": "a5d070e3-7f60-46e2-9722-16accacb8d9a", "children": ["683565cf-0205-47d1-85df-10a4bdd03d94"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: cannot find msedge binary\nStacktrace:\nBacktrace:\n\tMicrosoft::Applications::Events::EventProperties::unpack [0x00007FF7F3834812+26370]\n\tMicrosoft::Applications::Events::GUID_t::GUID_t [0x00007FF7F3789E02+339106]\n\tOrdinal0 [0x00007FF7F3317821+620577]\n\tOrdinal0 [0x00007FF7F333DB21+776993]\n\tOrdinal0 [0x00007FF7F333A8E8+764136]\n\tOrdinal0 [0x00007FF7F3374B4F+1002319]\n\tOrdinal0 [0x00007FF7F336F263+979555]\n\tOrdinal0 [0x00007FF7F3346070+811120]\n\tOrdinal0 [0x00007FF7F3347668+816744]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7F35E32C8+128088]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7F35CCC1B+36267]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7F35CFC3C+48588]\n\tMicrosoft::Applications::Events::EventProperty::to_string [0x00007FF7F3427973+22115]\n\tMicrosoft::Applications::Events::GUID_t::GUID_t [0x00007FF7F379142B+369355]\n\tMicrosoft::Applications::Events::GUID_t::GUID_t [0x00007FF7F37962C4+389476]\n\tMicrosoft::Applications::Events::GUID_t::GUID_t [0x00007FF7F379641D+389821]\n\tMicrosoft::Applications::Events::time_ticks_t::time_ticks_t [0x00007FF7F37A0EA9+40761]\n\tBaseThreadInitThunk [0x00007FFED9CB1412+34]\n\tRtlUserThreadStart [0x00007FFEDA9F5504+52]\n\n", "trace": "  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"G:\\pythonProject\\CredKart_Pytest_Project\\testCases\\conftest.py\", line 26, in setup\n    driver = webdriver.Edge()\n             ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\edge\\webdriver.py\", line 49, in __init__\n    super().__init__(\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 54, in __init__\n    super().__init__(\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1693548696182, "stop": 1693548756729}], "start": 1693548696182, "stop": 1693548757699}